src/btree/bt_split.c

```diff
@@ -1259,8 +1235,7 @@ __split_internal_lock(
 		parent = ref->home;
 
 		/* Encourage races. */
-		__page_split_timing_stress(
-		    session, WT_TIMING_STRESS_SPLIT_7, WT_THOUSAND);
```

Comment:
Did you deliberately change the sleep time for this option? I presume the sleep was very short here because it encouraged more races. How would you feel about adding a flags to `__wt_timing_stress` with a `SHORT` option?

Reply:
I did change it deliberately, I couldn't think of a reason the lock stress timer would be different than any other stress timer, and I didn't find anything in the history of the changes that made me think there was something interesting happening. If there's any evidence that a shorter stress timer here contributes to finding races, we should change it back, of course.

If we do change it back, I'd suggest a "fixed-number-of-usecs" argument to `__wt_timing_stress()`, because then the checkpoint timer stress code can use it too.